bump_version.py

```diff
@@ -150,8 +137,6 @@ def bump_file_list(file_list, regex_replacement):
 bump_file_list(pkginfo_files,pkginfo_files_dict)
 bump_file_list(pkgproj_files,pkgproj_files_dict)
 bump_file_list(test_files,test_files_dict)
```

Comment:
As the `test_files` variable was removed, shouldn't the `test_files` reference be removed?

Reply:
Not sure to remove the mentioned lines, because the `test_files_dict` dictionary was not removed. This dictionary contains the `wazuh_version` variable, used in some scripts of packages generation.